docs_src/microservices/device/V3Migration.md

```diff
@@ -8,29 +8,145 @@ See [Top Level V3 Migration Guide](../../../V3TopLevelMigration) for details app
 
 ### Device Files
 
-TBD
+1. Change device definition file to YAML format.
+2. Remove `LastConnected` and `LastReported` configs.
+3. ProtocolProperties now supports typed values.
+
+    !!! example "ProtocolProperty with typed values"
+    
+        ```yaml
+        protocols:
+          other:
+            Address: simple01
+            Port: 300
+        ```
 
 ### Device Profile Files
 
-TBD
+N/A
```

Comment:
There are several fields changed and added in the Device Profile.
Please refer to https://docs.edgexfoundry.org/3.0/V3TopLevelMigration/#core-metadata

Reply:
updated